the/mesh.html

```diff
@@ -108,7 +108,61 @@ <h2 class="to-animate">Meet the Mesh</h2>
             <div class="grid-container">
               <div class="grid-element">
                 <figure class="circular-landscape">
-                  <img src="./images/The-Mesh/Alex-Xiang.jpg" alt="Alex" />
+                  <img src="../images/The-Mesh/Ralph-Lavelle.jpg" alt="Ralph" />
+                </figure>
+                <div class="div-text">
+                  <p class="mesh-name">Ralph</p>
+                  <p class="mesh-title">Sudo Developer</p>
+                </div>
+              </div>
+              <div class="grid-element">
+                <figure class="circular-landscape">
+                  <img src="../images/The-Mesh/Barry-Easman.jpg" alt="Barry" />
+                </figure>
+                <div class="div-text">
+                  <p class="mesh-name">Barry</p>
+                  <p class="mesh-title">Sudo Developer</p>
+                </div>
+              </div>
+              <div class="grid-element">
+                <figure class="circular-landscape">
+                  <img src="../images/The-Mesh/Preston-Harrison.jpg" alt="Preston" />
+                </figure>
+                <div class="div-text">
+                  <p class="mesh-name">Preston</p>
+                  <p class="mesh-title">Developer</p>
+                </div>
+              </div>
+              <div class="grid-element">
+                <figure class="circular-landscape">
+                  <img src="../images/The-Mesh/Taylor-Scheel.jpg" alt="Taylor" />
+                </figure>
+                <div class="div-text">
+                  <p class="mesh-name">Taylor Scheel</p>
```

Comment:
Some members have just first name, others have both first and last names. Any reasoning behind that?

Reply:
Resolved